src/bus.jl

```diff
@@ -36,14 +36,14 @@ function LoadBus(coords::LatLon, load, volt, pop, connected_to = Set(), connecti
     return LoadBus(-1, coords, load, volt, pop, connected_to, connections)
 end
 
-function LoadBus{T <: Real}(
+function LoadBus(
         coords::Tuple{T,T},
```

Comment:
If `T` is not used in the function body you should be able to write this as:
```suggestion
        coords::Tuple{T,T} where T <: Real,
```